django/contrib/postgres/constraints.py

```diff
@@ -77,6 +78,15 @@ def _get_expressions(self, schema_editor, query):
             expressions.append(expression)
         return ExpressionList(*expressions).resolve_expression(query)
 
+    def _check(self, model, connection):
+        references = set()
+        for expr, _ in self.expressions:
+            if isinstance(expr, str):
+                references.add(expr.split(LOOKUP_SEP))
```

Comment:
I would wrap in `F()` and use `_get_expr_references()` to get the same references for `F(lookup_string)` and `lookup_string`. Now, we have `list` vs. `tuple`.